frontend/src/metabase/query_builder/components/VisualizationError/VisualizationError.tsx

```diff
@@ -25,19 +24,9 @@ import {
   QueryErrorMessage,
   QueryErrorContent,
 } from "./VisualizationError.styled";
+import { AdminEmail } from "./components";
 import { adjustPositions, stripRemarks } from "./utils";
 
-function EmailAdmin(): JSX.Element | null {
-  const hasAdminEmail = isNotNull(MetabaseSettings.adminEmail());
-  return hasAdminEmail ? (
-    <span className={QueryBuilderS.QueryErrorAdminEmail}>
-      <a className={CS.noDecoration} href={`mailto:${hasAdminEmail}`}>
-        {hasAdminEmail}
```

Comment:
The bug was that a boolean was used instead of a string, I think someone confused `isNotNull` with `checkNotNull`.